packages/date-picker/theme/material/vaadin-date-picker-overlay-content-styles.js

```diff
@@ -113,11 +113,6 @@ registerStyles(
       background: var(--material-background-color);
     }
 
-    :host([years-visible]) [part='years']::after {
-      top: calc(20px + 16px);
-      height: calc(100% - 20px - 16px);
-    }
```

Comment:
The `::after` pseudo-element was removed in https://github.com/vaadin/vaadin-material-styles/commit/020e32611054c293862d64efeb9f5fdda800983a while polishing the Material theme before adding it to the component, so these leftover styles are unused.